packages/terra-date-picker/src/react-datepicker/date_utils.js

```diff
@@ -67,7 +67,13 @@ export function cloneDate (date) {
 
 export function parseDate (value, { dateFormat, locale }) {
   const m = moment(value, dateFormat, locale || moment.locale(), true)
-  return m.isValid() ? m : null
+
+  // Only support dates between 01/01/1900 and 12/31/2010 because the year dropdown only allows 1900 - 2100.
+  if (m.isValid() && m.isAfter('1900-01-01') && m.isBefore('2100-12-31')) {
+    return m;
+  }
```

Comment:
Is it possible to use DateUtil.isBetweenMinMaxDate for this check?

Reply:
Definitely. I added this before creating DateUtil.isBetweenMinMaxDate since it is needed in multiple places and forgot to come back here to use it.
https://github.com/cerner/terra-framework/pull/1132/commits/f37f71c9d677bf16a39b3ee2fb62494cbd0c6c25